src/main/webapp/destinationCreation.js

```diff
@@ -22,31 +32,87 @@ function addScriptToHead() { // eslint-disable-line
 }
 
 function searchForPlace() { // eslint-disable-line
+  const element = document.querySelectorAll('#place');
+  if (element.length > 0) {
+    let i;
+    for (i = 0; i < element.length; i++) {
```

Comment:
You don't need to define i before the for loop
```suggestion
    for (let i = 0; i < element.length; i++) {
```